discovery-provider/src/tasks/index_nethermind.py

```diff
@@ -77,7 +77,7 @@
 # HELPER FUNCTIONS
 
 default_padded_start_hash = (
```

Comment:
do we use `default_padded_start_hash` anywhere anymore?

Reply:
tbh idk what that is for. only for 
```
            # Special case for default start block value of 0x0 / 0x0...0
            if revert_block.parenthash == default_padded_start_hash:
                parent_hash = default_config_start_hash
```
i think this is meant for local